src/Command/GenerateObjectAclCommand.php

```diff
@@ -97,6 +98,7 @@ public function execute(InputInterface $input, OutputInterface $output)
                 continue;
             }
 
+            assert($admin instanceof AdminInterface);
```

Comment:
Should we use `assert`s in general? Symfony is not using asserts in any of its code.

Reply:
I've tried it at work, and I think it's slightly better than comments because:

- it might get checked if a dev has assertions enabled
- it can be used to have intersection types without resorting to not-fully-supported-everywhere phpdoc like `/** @var A&B */`
More on this here: https://github.com/doctrine/coding-standard/pull/47

Reply:
Also, the Travis build fails because this assertion is incorrect, which shows why assertions are better ;)

Reply:
Turns out this one is very very hard to fix, I'll just remove it for now, but there's something quite wrong in there: the container throws on missing service, while our tests mock it so that it returns null…

Reply:
Ah it's not this one, it's the one in Pool::getInstance, sorry.